### ISSUE TITLE
[Bug] [a11y] ETP radio buttons labels on the onboarding screen incorrect reading order

### ISSUE THREAD

**Issue by sv-ohorvath** [mozilla-mobile/fenix#8627]
## Steps to reproduce
1. Clear app data. Open app.
2. On the onboarding screen, got o the ETP card.
3. Select to read the 2 radio buttons: Strict & Standard.
### Expected behavior
Standard and Strict labels should be announced first when each is selected.
### Actual behavior
The entire element is selected and no label is announced, only "double-tap to activate". Swipping right to go to the next in the list will eventually read the radio button and its label.
### Device information

* Android device: Galaxy Tab 3 (Android 8)
* Fenix version: Beta 4.0-build1

**Comment by sv-sdeiac** [comment]
Verified as fixed using the latest version of Firefox Preview Beta 4.0.0-beta.2.

Device:
- Nexus 9 (Android 7.1.1).
